src/uglify/serialization.js

```diff
@@ -0,0 +1,46 @@
+const toType = value => (Object.prototype.toString.call(value).slice(8, -1));
```

Comment:
The Thank you for discovering this question I now think of three viable options:

1. When `parallel.workers> 0`, users are not allowed to use` Function` or` RegExp`
2. Before using serialization, use ESlint to check the "function" and report the "function" of context-dependent
3. At the run-time stage, catch the error and tell the user why the error occurred

I am currently inclined to the third program because it is friendly and simple. 😊